javascripts/discourse/components/tag-game-card-contents.gjs

```diff
@@ -39,20 +52,60 @@ export default class TagGameCardContents extends Component {
   async fetchTagData() {
     const { data } = await graphql(TAG_QUERY, { slug: this.args.data.tag });
 
-    this.tagData = data.product[0];
+    this.priceData = data.product_price[0] || null;
+    this.tagData = this.priceData.product || null;
 
     if (!this.tagData) {
       this.router.transitionTo("tag.show", this.args.data.tag);
     }
+  }
 
-    // TODO remove this
-    this.tagDataJson = JSON.stringify(this.tagData, null, 2);
+  get tagDataJson() {
```

Comment:
I don't think we need the stringified JSON anymore

Reply:
Removed